heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+''' argparser.py: HERON RC parser support for specifying config level arguments in an RC file.'''
+import argparse
+import collections
+import json
+import os
+import re
+import sys
+import traceback
+from heron.common.src.python.color import Log
+import heron.common.src.python.utils.config as config
+
+
+##########################################################################
+# Run the command
+##########################################################################
+
+HERON_RC_FILE = "~/.heronrc"
```

Comment:
good point, i will combine : HERON_RC_FILE = "~/.heronrc"
HERON_RC = os.path.expanduser(HERON_RC_FILE)

In addition, you may want to note a couple of things:
1. there was supposed to be a system level heron rc (under /etc/)
2. Karthik mentioned that there may be an implicit hierarchy so that at a system level , there could be heron options, and at a user level , all future additions
3. the arg parser currently can also be initialized with an RC file , so the one under home is the default.